.github/workflows/dryrun.yml

```diff
@@ -0,0 +1,57 @@
+# /********************************************************************************
+# * Copyright (c) 2022 Contributors to the Eclipse Foundation
+# *
+# * See the NOTICE file(s) distributed with this work for additional
+# * information regarding copyright ownership.
+# *
+# * This program and the accompanying materials are made available under the
+# * terms of the Apache License 2.0 which is available at
+# * https://www.apache.org/licenses/LICENSE-2.0
+# *
+# * SPDX-License-Identifier: Apache-2.0
+# ********************************************************************************/
+
+name: BitBake DryRun
+on:
+  push:
+  workflow_dispatch:
+
+jobs:
+  main:
+    name: Yocto (Master)
```

Comment:
we only have main not master (not to be confused)

Reply:
The "(Master)" indicates the this config is working against Yocto Master branches (they call them master).
It's not our branch name, but the branch name of Yocto